modern/pnpm-lock.yaml

```diff
@@ -0,0 +1,13362 @@
+lockfileVersion: 5.4
```

Comment:
We don't include lock files in the repo. Please remove and add to the ignore file.

Reply:
Done.